src/main/kotlin/notary/btc/BtcNotaryInitialization.kt

```diff
@@ -12,14 +12,19 @@ import org.bitcoinj.core.PeerGroup
 import org.bitcoinj.params.RegTestParams
 import org.bitcoinj.store.LevelDBBlockStore
 import org.bitcoinj.wallet.Wallet
+import provider.NotaryPeerListProviderImpl
 import provider.btc.BtcRegisteredAddressesProvider
 import sidechain.SideChainEvent
+import sidechain.iroha.util.ModelUtil
 import java.io.File
 
 class BtcNotaryInitialization(
     private val btcNotaryConfig: BtcNotaryConfig,
     private val btcRegisteredAddressesProvider: BtcRegisteredAddressesProvider
 ) {
+
+    val keypair = ModelUtil.loadKeypair(btcNotaryConfig.iroha.pubkeyPath, btcNotaryConfig.iroha.privkeyPath).get()
```

Comment:
Why get()? Why not map() or at least flatMap()

Reply:
Get extracts value from result in success case and throws an exception if here is failure. It is enough for the test.